### PR TITLE
Roaming enemy stuff

### PR DESCRIPTION
**Pull request by BlinkingBerry**
Hello. I tried to do some of that "giving labels and stuff actual names". In parts relating to roaming enemies. I hope this is okay.

### CODE REVIEW

File: src/code/entities/bank3.asm [hunk 1]
```diff
@@ -9335,38 +9372,43 @@ GetEntityYDistanceAwayFromLink::
     ld   d, a                                     ; $7EFC: $57
     ret                                           ; $7EFD: $C9
 
-func_003_7EFE::
+; Inputs:
+;   bc   entity index
+;
+; Outputs:
+;   e   entity's direction to Link
```
Comment (daid) [inline src/code/entities/bank3.asm:9379]: 0 = right 1 = left 2 = up, 3 is down?
Might be nice to document that, or if there are some consts that are used for this.

**Comment by daid** [comment]
Overall, looks good and more clear then what we had.
Weird that `GetEntityDirectionToLink_trampoline` isn't used anywhere...

**Review by daid — APPROVED** [review]

File: src/code/entities/bank3.asm (hunk 1 above)
Comment (BlinkingBerry) [reply src/code/entities/bank3.asm]: There are the DIRECTION_* constants. I've edited the comment to mention them.

**Comment by kemenaran** [comment]
Forgot to merge this… Thanks for the PR!

**Pull request merged**